src/lib/source-handlers/azure/list-projects.ts

```diff
@@ -75,18 +76,13 @@ async function getProjects(
     'api-version': '4.1',
   };
   const query = qs.stringify(params);
-  const data = await limiter.schedule(() =>
-    needle(
-      'get',
-      `${host}/${orgName}/_apis/projects?${query}`, //document the version somewhere
-      {
-        headers: { Authorization: 'Basic ' + base64.encode(':' + azureToken) },
-      },
-    ),
+  const headers: OutgoingHttpHeaders = { Authorization: `Basic ${base64.encode(':' + azureToken)}` }
+  const data = await requestWithRateLimitRetries(
+    'get',
+    `${host}/${orgName}/_apis/projects?${query}`,
+    headers,
+    limiter
   );
-  if (data.statusCode != 200) {
```

Comment:
you need to keep this, only the `const = data` line should change. Even after 7 retries it can be a non 200 status code so we need to handle it

Reply:
Done